source/manual/review-apps.html.md

```diff
@@ -0,0 +1,166 @@
+---
+owner_slack: '#2ndline'
+review_by: 2017-05-20
+title: Review apps for Pull Requests
+section: Deployment
+layout: manual_layout
+parent: "/manual.html"
+---
+
+# Review apps for Pull Requests
+
+Review apps allow us to automatically deploy application changes into a brand
+new Heroku app so that each Pull Request reviewer has a chance to see how it
+looks with live data.
+
+There are currently 2 examples of GOV.UK apps that have review apps enabled:
+- [collections](https://github.com/alphagov/collections)
+- [government-frontend](https://github.com/alphagov/government-frontend)
+
+Follow the steps below in order to enable review apps for your GOV.UK
+application.
+
+## Create a Heroku account
+
+If you don't have a Heroku account yet, please create one using your work email
+address.
+
+You can do so on this page: https://signup.heroku.com
+
+## Install the Heroku Toolbelt
```

Comment:
We probably can, though I haven't done it that way. I can delete this if it's preferable.